Source/WebKit/_WebKit_SwiftUI/API/View+WebViewModifiers.swift

```diff
@@ -101,7 +100,15 @@ extension View {
 #endif
     }
 
-    @_spi(Private)
+    /// Specifies the visibility of the webpage's natural background color within this view.
+    ///
+    /// By default, WebViews are opaque, and use the page's natural background color as their background color. Use this modifier if you would like to
```

Comment:
I noticed we are generally inconsistent in this file about using "web view(s)", "WebView(s)", and whether or not "WebView" is code-formatted. Maybe something to clean up later.